lib/src/util.dart

```diff
@@ -42,25 +42,25 @@ class Util {
   }
 
   static Uint8List read(int bytes, UtilFunc<ffi.Uint8> readFunc) {
-    final ptr = ffi.allocate<ffi.Uint8>(count: bytes);
+    final ptr = ffi.calloc<ffi.Uint8>(bytes);
     final len = call(() => readFunc(ptr));
     final res = Uint8List.fromList(ptr.asTypedList(len));
-    ffi.free(ptr);
+    ffi.calloc.free(ptr);
     return res;
   }
 
   static int write(Uint8List bytes, UtilFunc<ffi.Uint8> writeFunc) {
     final len = bytes.length;
-    final ptr = ffi.allocate<ffi.Uint8>(count: len);
+    final ptr = ffi.calloc<ffi.Uint8>(len);
     ptr.asTypedList(len).setAll(0, bytes);
     final res = call(() => writeFunc(ptr));
-    ffi.free(ptr);
+    ffi.calloc.free(ptr);
     return res;
   }
 
   static String? fromUtf8(ffi.Pointer<ffi.Int8> str) {
     if (str == ffi.nullptr) return null;
-    final length = ffi.Utf8.strlen(str.cast());
+    final length = ffi.Utf8Pointer(str.cast()).length; // ffi.Utf8.strlen(str.cast());
```

Comment:
could you clean up the comment?

Reply:
I use lib prefix for win libs too) and wanted to leave the option to force the prefix) I removed this

Reply:
Oh, ok. For a moment I thought the lib prefix was wrong on Windows. Anyway, I think we could make this doable with dylib.dart